src/toil/fileStores/abstractFileStore.py

```diff
@@ -447,11 +449,12 @@ def open(cls, outer=None):
             raise NotImplementedError()
 
         @classmethod
-        def _load(cls, fileName):
+        def _load(cls, fileName: str) -> Any: 
```

Comment:
I tried returning `'_StateFile'` here instead but it kept saying that it is not defined. Using `from __future__ import annotations` to postpone type checking or creating a generic variable: `T = TypeVar('T', bound='_StateFile')` didn't seem to be working either.